src/api/departures/schema.ts

```diff
@@ -71,41 +80,33 @@ export const postDeparturesRequest = {
 };
 
 export const getQuayDeparturesRequest = {
-  query: Joi.object({
+  query: Joi.object<QuayDeparturesQueryVariables>({
     id: Joi.string().required(),
     numberOfDepartures: Joi.number().default(10),
     startTime: Joi.string(),
     timeRange: Joi.number().default(86400),
-    numberOfDeparturesPerLineAndDestinationDisplay: Joi.number(),
+    limitPerLine: Joi.number(),
```

Comment:
Wrong name for parameter was used here. Referenced as `limitPerLine` everywhere else, so it did work, but now it checks that input is actually a number.